### PR TITLE
Adding content for October, November, December

### PR DESCRIPTION
**Pull request by RabeaGleissner**
Hi ladies, 
Could you double-check if everything is correct? I didn't attend the October and November meetups, so I kind of combined the content from the meetup pages and what you've been saying about those meetups. Hopefully it's all correct!
Any typos or weird language, let me know and I'll update the PR.
Otherwise, please merge! @iteles @NataliaLKB

### CODE REVIEW

File: meetups/2016-october/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The other event was a ThingMonk conference for developers with an embedded hackday where she built a voice activated light colour changing connected device.
+
+If you measure success by prizes and gifts, then in September she received an amazon echo, raspberry pi, amazon vouchers, IoT books and more. However the real prize was the experience of being there, the connections she made and the support she received from her team, the event volunteers, the mentors, the coaches and the attendees!
+
+###How to Accidentally Start a Business in 54 Hours - presented by [_Claire Mitchell_](https://twitter.com/nofootnotes) and [_Michelle Garrett_](https://twitter.com/msmichellegar)
+
+Claire and Michelle won [Startup Weekend](https://startupweekend.org/) last year, launching them on an exciting and tumultuous journey that took them to Poland and back. They’ll be sharing their experiences turning a weekend project into a real business, and their tips for hackathon success.
```
Comment (inespons) [inline meetups/2016-october/readme.md]: We need to change the tense so this sits in the past.

Comment (RabeaGleissner) [inline meetups/2016-october/readme.md]: Oops! Good spot, thanks @inespons
I'll update it tomorrow morning, no need for you to do it. Should be quick, I think.

**Comment by RabeaGleissner** [comment]
@inespons Hi Ines! I updated the wording to past tense. Hope everything is ok now. Could you merge the PR? Thanks!

**Comment by RabeaGleissner** [comment]
@inespons Hi Ines! I updated the wording to past tense. Hope everything is ok now. Could you merge the PR? Thanks!

**Comment by inespons** [comment]
Great, cheers!
:shipit:

**Pull request merged**